scss/_footer.scss

```diff
@@ -0,0 +1,30 @@
+footer {
+  background-color: $color-white-wrapper;
+  padding: 2rem 0;
+  .row {
+    justify-content: space-around;
+    .footer__privacy {
+      display: flex;
+      justify-content: center;
+
+      a {
+        color: $color-black;
+        text-decoration: none;
+        padding: 0 0.25rem;
+        span {
+        }
+      }
+    }
+    .footer__social-media {
```

Comment:
Jeśli budujesz tak selectory to nie ma potrzeby powtarzać członu `.footer`, zamiast tego możesz użyć `&`, czyli:

`&__social-media {}`

`&` - powtarza wszystkie parent selectory